crates/ruff_linter/src/rules/pydocstyle/rules/indent.rs

```diff
@@ -234,17 +241,22 @@ pub(crate) fn indent(checker: &mut Checker, docstring: &Docstring) {
 
     if checker.enabled(Rule::OverIndentation) {
         // If every line (except the last) is over-indented...
-        if is_over_indented {
-            for over_indented in over_indented_lines {
+        if is_over_indented && !over_indented_lines.is_empty() {
```

Comment:
Is the second condition necessary? Since we won't iterate below anyway if empty, right?

Reply:
Yes correct. This solved an offset underflow issue I was running into when things were structured differently.
```suggestion
        if is_over_indented {
```